test/HubPool.PooledTokenSynchronization.ts

```diff
@@ -33,7 +33,9 @@ describe("HubPool Pooled Token Synchronization", function () {
 
     // Execute a relayer refund. Check counters move accordingly.
     const { tokensSendToL2, realizedLpFees, leafs, tree } = await constructSingleChainTree(weth);
-    await hubPool.connect(dataWorker).initiateRelayerRefund([3117], 1, tree.getHexRoot(), consts.mockTreeRoot);
+    await hubPool
+      .connect(dataWorker)
+      .initiateRelayerRefund([3117], 1, tree.getHexRoot(), consts.mockTreeRoot, consts.mockSlowRelayFulfillmentRoot);
```

Comment:
here is another example of the long ass variable names and how it makes the code ugly.

Reply:
Yeah, we can update these when we update the other names. I agree that shorter names would be better! So let's decide offline what these should be called and do a big naming PR.